pkgs/development/python-modules/open-hypergraphs/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  numpy,
+  scipy,
+  hypothesis,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "open-hypergraphs";
+  version = "0.1.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "statusfailed";
+    repo = "open-hypergraphs";
+    tag = "pypi-${version}";
+    hash = "sha256-ifcQXZDnOvo2XL7WYVFLv2iHWhImUSp3jqAPPYySNjU=";
+  };
+
+  build-system = [
+    hatchling
+  ];
+
+  dependencies = [
+    numpy
+    scipy
+  ];
+
+  pythonRelaxDeps = [
+    "numpy"
+    "scipy"
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    hypothesis
+  ];
+
+  pythonImportsCheck = [
+    "open_hypergraphs"
+  ];
+
+  meta = {
+    description = "An implementation of open hypergraphs for string diagrams";
```

Comment:
I don't think the description fits guidelines

Reply:
Indeed.

Reply:
Is your concern that I don't use the even more esoteric "about" from the github repo ("a datastructure for scalable combinatorial syntax") or that the package implements open hypergraphs and is also named open hypergraphs?

Reply:
Take a look at this: https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#description-var-meta-description

Moreso that it beings with a preposition

Reply:
Good catch, fixed.